app/View/CoPeople/expunge.ctp

```diff
@@ -192,16 +192,26 @@
         'span',
         '',
         array(
-          'class' => 'spinner-grow spinner-grow-sm mr-1 align-middle invisible btn-submit-with-loader',
+          'class' => 'spinner-grow spinner-grow-sm mr-2 align-middle invisible btn-submit-with-loader',
           'escape' => false,
           'role' => 'status',
           'aria-hidden' => true
```

Comment:
Set the aria-hidden attribute to 'true' (string) rather than true (= 1 in output). The attribute can only take the string value.